### ISSUE TITLE
Questions re: sample csv

### ISSUE THREAD

**Issue by ribose-jeffreylau** [riboseinc/structured_csv#3]
In one of the sample csv files, the following line contains a possible issue:

```csv
DATA,,,,,,,,,,,,,,
geographic_area.fr,geographic_area.en,geographic_area.es,restrictions.telex_not_provided[boolean],<snip>
Andorre,Andorra,Andorra,TRUE,<snip>
```

The uppercase `TRUE` is currently not recognized as type `boolean` which is what is suggested here in the header line.  Is this intended, or should boolean values match their upper-cased versions as well?

**Comment by ronaldtse** [comment]
Boolean values should also match upper-cased versions, so it's a bug.

**Comment by ribose-jeffreylau** [comment]
Should they match `TrUe`, `True`, `trUE`, etc., or just `TRUE` and `true`?

**Comment by ribose-jeffreylau** [comment]
Also, would you allow surrounding spaces to be trimmed? like, `    true     ` to be interpreted as `true`?

**Comment by ronaldtse** [comment]
@ribose-jeffreylau it's all up to you? 😉

**Comment by ribose-jeffreylau** [comment]
👌  I'll make it so it trims whitespaces and matches all cases then.

**Comment by ronaldtse** [comment]
Thanks @ribose-jeffreylau !

**Comment by ribose-jeffreylau** [comment]
Closed in ad9d88c84e8248c5ff561091d73bf8ca2c0a86c0
